lock-api/src/test/java/com/palantir/lock/client/AsyncTimeLockUnlockerTest.java

```diff
@@ -1,11 +1,11 @@
 /*
- * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
```

Comment:
Checkstyle sent this file back in time

Reply:
Yep. I think when I was fixing checkstyle I decided to copy it from the first file I could find that had the right version 🕙 changed